azure.go

```diff
@@ -270,11 +283,40 @@ func (ac *AzureClient) listByTag(tagName string, tagValue string) ([]string, err
 		return nil, fmt.Errorf("Error unmarshalling response body: %v", err)
 	}
 
-	resources := extractResourceNames(data, subscription)
-
+	var resources []AzureResource
```

Comment:
So you removed the extractResourceName method ? Do you think we could refactor that with a method ?

Reply:
I eventually added a method to extend the `AzureResource` type instead

Reply:
Nice !